gradle-baseline-java/src/test/groovy/com/palantir/baseline/plugins/BaselineImmutablesTest.groovy

```diff
@@ -16,10 +16,13 @@
 
 package com.palantir.baseline.plugins
 
+import com.google.common.collect.ImmutableList
 import nebula.test.IntegrationSpec
+import nebula.test.functional.ExecutionResult
 
 class BaselineImmutablesTest extends IntegrationSpec {
     private static final String IMMUTABLES = 'org.immutables:value:2.8.8'
+    private static final String IMMUTABLES_ANNOTATIONS = 'org.immutables:value:2.8.8:annotations'
```

Comment:
Curious when we bump these to test compatibility (e.g. latest is 2.9.2)

Reply:
I ran tests with both versions, 2.9.2 seems solid where I’ve tried it, but this week has been support-heavy so I’d prefer not to tempt fate quite yet :-)

Reply:
:+1: Definitely understand